diff -up falkon-3.1.0/src/main/main.cpp.3~ falkon-3.1.0/src/main/main.cpp
--- falkon-3.1.0/src/main/main.cpp.3~	2019-05-11 17:15:24.193255861 +0200
+++ falkon-3.1.0/src/main/main.cpp	2019-05-11 17:22:48.771140375 +0200
@@ -23,6 +23,8 @@
 #include <iostream>
 
 #if defined(Q_OS_LINUX) || defined(__GLIBC__) || defined(__FreeBSD__) || defined(__HAIKU__)
+#include <unistd.h>
+#include <sys/types.h>
 #include <signal.h>
 #include <execinfo.h>
 
@@ -142,6 +144,17 @@ int main(int argc, char* argv[])
     }
     delete socket;
 
+    if(getuid() == 0 || geteuid() == 0) {
+	    // QtWebEngine with sandboxing can't run as root
+	    // https://bugs.chromium.org/p/chromium/issues/detail?id=638180
+	    // So we need to either switch to another user, or
+	    // disable sandboxing.
+	    // Since a user might be trying to download stuff to
+	    // a root-owned directory when recovering a system etc.,
+	    // let's go for the latter...
+	    qputenv("QTWEBENGINE_DISABLE_SANDBOX", QByteArray::number(1));
+    }
+
     // Hack to fix QT_STYLE_OVERRIDE with QProxyStyle
     const QByteArray style = qgetenv("QT_STYLE_OVERRIDE");
     if (!style.isEmpty()) {
