diff -up falkon-3.1.0/src/plugins/VerticalTabs/verticaltabsplugin.cpp.omv~ falkon-3.1.0/src/plugins/VerticalTabs/verticaltabsplugin.cpp
--- falkon-3.1.0/src/plugins/VerticalTabs/verticaltabsplugin.cpp.omv~	2019-12-27 01:27:54.760398580 +0100
+++ falkon-3.1.0/src/plugins/VerticalTabs/verticaltabsplugin.cpp	2019-12-27 01:28:01.254398265 +0100
@@ -31,6 +31,7 @@
 #include "desktopfile.h"
 
 #include <QSettings>
+#include <QFile>
 
 VerticalTabsPlugin::VerticalTabsPlugin()
     : QObject()
