diff -up falkon-24.01.80/src/lib/app/browserwindow.cpp.5~ falkon-24.01.80/src/lib/app/browserwindow.cpp
--- falkon-24.01.80/src/lib/app/browserwindow.cpp.5~	2023-11-28 03:10:55.000000000 +0100
+++ falkon-24.01.80/src/lib/app/browserwindow.cpp	2023-12-10 23:39:55.836379774 +0100
@@ -573,7 +573,7 @@ void BrowserWindow::loadSettings()
 
     //Url settings
     settings.beginGroup(QSL("Web-URL-Settings"));
-    m_homepage = settings.value(QSL("homepage"), QSL("falkon:start")).toUrl();
+    m_homepage = settings.value(QSL("homepage"), QSL("falkon:speeddial")).toUrl();
     settings.endGroup();
 
     //Browser Window settings
diff -up falkon-24.01.80/src/lib/data/data/bookmarks.json.5~ falkon-24.01.80/src/lib/data/data/bookmarks.json
--- falkon-24.01.80/src/lib/data/data/bookmarks.json.5~	2023-11-28 03:10:55.000000000 +0100
+++ falkon-24.01.80/src/lib/data/data/bookmarks.json	2023-12-10 23:39:55.836379774 +0100
@@ -1,7 +1,51 @@
 {
  "roots": {
   "bookmark_bar": {
-   "children": [ ],
+   "children": [
+       {
+            "description": "OpenMandriva",
+            "keyword": "openmandriva",
+            "name": "OpenMandriva",
+            "type": "url",
+            "url": "https://www.openmandriva.org/",
+            "visit_count": 0
+        },
+        {
+            "description": "OpenMandriva forums",
+            "keyword": "openmandriva-forum",
+            "name": "OpenMandriva Forums",
+            "type": "url",
+            "url": "https://forum.openmandriva.org/",
+            "visit_count": 0
+        },
+        {
+            "description": "OpenMandriva issues tracker",
+            "keyword": "openmandriva-issues",
+            "name": "File a bug report",
+            "type": "url",
+            "url": "https://github.com/OpenMandrivaAssociation/distribution/issues/",
+            "visit_count": 0
+        },
+        {
+            "type": "separator"
+        },
+        {
+            "description": "OpenMandriva Wikipedia",
+            "keyword": "openmandriva-wiki",
+            "name": "OpenMandriva Wiki",
+            "type": "url",
+            "url": "https://wiki.openmandriva.org/",
+            "visit_count": 0
+        },
+        {
+            "description": "OpenMandriva Automatic Build Farm",
+            "keyword": "openmandriva-abf",
+            "name": "OpenMandriva Automatic Build Farm",
+            "type": "url",
+            "url": "https://abf.openmandriva.org/",
+            "visit_count": 0
+        }
+    ],
    "description": "Bookmarks located in Bookmarks Toolbar",
    "expanded": true,
    "expanded_sidebar": true,
diff -up falkon-24.01.80/src/lib/preferences/preferences.cpp.5~ falkon-24.01.80/src/lib/preferences/preferences.cpp
--- falkon-24.01.80/src/lib/preferences/preferences.cpp.5~	2023-12-10 23:39:55.836379774 +0100
+++ falkon-24.01.80/src/lib/preferences/preferences.cpp	2023-12-10 23:41:50.356659441 +0100
@@ -135,7 +135,7 @@ Preferences::Preferences(BrowserWindow*
     Settings settings;
     //GENERAL URLs
     settings.beginGroup(QSL("Web-URL-Settings"));
-    m_homepage = settings.value(QSL("homepage"), QUrl(QSL("falkon:start"))).toUrl();
+    m_homepage = settings.value(QSL("homepage"), QUrl(QSL("file:///usr/share/mdk/indexhtml/index.html"))).toUrl();
     m_newTabUrl = settings.value(QSL("newTabUrl"), QUrl(QSL("falkon:speeddial"))).toUrl();
     ui->homepage->setText(QString::fromUtf8(m_homepage.toEncoded()));
     ui->newTabUrl->setText(QString::fromUtf8(m_newTabUrl.toEncoded()));
@@ -420,6 +420,15 @@ Preferences::Preferences(BrowserWindow*
     m_notifPosition = settings.value(QSL("Position"), QPoint(10, 10)).toPoint();
     settings.endGroup();
 
+    //SPEEDDIAL
+    settings.beginGroup(QSL("SpeedDial"));
+    settings.setValue(QSL("background"), QSL("file:///usr/share/wallpapers/default.png"));
+    settings.setValue(QSL("backsize"), QSL("cover"));
+    settings.setValue(QSL("pages"), QSL("url:\"https://openmandriva.org\"|title:\"OpenMandriva\";url:\"https://forum.openmandriva.org\"|title:\"OpenMandriva Forum\";url:\"https://abf.openmandriva.org\"|title:\"Automatic Build Farm\";"));
+    settings.setValue(QSL("pagesrow"), QSL("4"));
+    settings.setValue(QSL("sdcenter"), QSL("true"));
+    settings.endGroup();
+
     //SPELLCHECK
     settings.beginGroup(QSL("SpellCheck"));
     ui->spellcheckEnabled->setChecked(settings.value(QSL("Enabled"), false).toBool());
