diff -up falkon-25.04.0/src/lib/preferences/preferences.cpp.4~ falkon-25.04.0/src/lib/preferences/preferences.cpp
--- falkon-25.04.0/src/lib/preferences/preferences.cpp.4~	2025-04-11 05:09:57.000000000 +0200
+++ falkon-25.04.0/src/lib/preferences/preferences.cpp	2025-04-18 00:48:31.245323199 +0200
@@ -290,7 +290,7 @@ Preferences::Preferences(BrowserWindow*
     ui->animateScrolling->setChecked(settings.value(QSL("AnimateScrolling"), true).toBool());
     ui->wheelScroll->setValue(settings.value(QSL("wheelScrollLines"), qApp->wheelScrollLines()).toInt());
     ui->printEBackground->setChecked(settings.value(QSL("PrintElementBackground"), true).toBool());
-    ui->useNativeScrollbars->setChecked(settings.value(QSL("UseNativeScrollbars"), false).toBool());
+    ui->useNativeScrollbars->setChecked(settings.value(QSL("UseNativeScrollbars"), true).toBool());
     ui->disableVideoAutoPlay->setChecked(settings.value(QSL("DisableVideoAutoPlay"), false).toBool());
     ui->webRTCPublicIpOnly->setChecked(settings.value(QSL("WebRTCPublicIpOnly"), true).toBool());
     ui->dnsPrefetch->setChecked(settings.value(QSL("DNSPrefetch"), true).toBool());
diff -up falkon-25.04.0/src/lib/webengine/webscrollbarmanager.cpp.4~ falkon-25.04.0/src/lib/webengine/webscrollbarmanager.cpp
--- falkon-25.04.0/src/lib/webengine/webscrollbarmanager.cpp.4~	2025-04-11 05:09:57.000000000 +0200
+++ falkon-25.04.0/src/lib/webengine/webscrollbarmanager.cpp	2025-04-18 00:09:11.063431200 +0200
@@ -108,7 +108,7 @@ WebScrollBarManager::WebScrollBarManager
 
 void WebScrollBarManager::loadSettings()
 {
-    m_enabled = Settings().value(QSL("Web-Browser-Settings/UseNativeScrollbars"), false).toBool();
+    m_enabled = Settings().value(QSL("Web-Browser-Settings/UseNativeScrollbars"), true).toBool();
 
     if (!m_enabled) {
         for (WebView *view : m_scrollbars.keys()) {
