diff -up falkon-3.1.0/src/lib/tools/qztools.cpp.omv~ falkon-3.1.0/src/lib/tools/qztools.cpp
--- falkon-3.1.0/src/lib/tools/qztools.cpp.omv~	2020-03-24 20:23:42.859831900 +0100
+++ falkon-3.1.0/src/lib/tools/qztools.cpp	2020-03-24 20:23:46.659812246 +0100
@@ -25,6 +25,7 @@
 #include <QByteArray>
 #include <QPixmap>
 #include <QPainter>
+#include <QPainterPath>
 #include <QBuffer>
 #include <QFile>
 #include <QDir>
